File: src/main/java/sh/sagan/phobos/builder/buildpattern/impl/DefaultBuildPattern.java
package sh.sagan.phobos.builder.buildpattern.impl;

import com.sk89q.worldedit.extent.clipboard.Clipboard;
import org.bukkit.Location;
import sh.sagan.phobos.builder.buildpattern.BuildPattern;

import java.util.List;

public class DefaultBuildPattern implements BuildPattern {

    @Override
    public List<Location> arrange(List<Location> toArrange, Clipboard clipboard) {
        return toArrange;
    }
}

File: src/main/java/me/sagan/phobos/builder/buildpattern/impl/DefaultBuildPattern.kt
package me.sagan.phobos.builder.buildpattern.impl

import com.sk89q.worldedit.extent.clipboard.Clipboard
import com.sk89q.worldedit.math.BlockVector3
import me.sagan.phobos.builder.buildpattern.BuildPattern


class DefaultBuildPattern() : BuildPattern(Direction.X_AXIS) {

    override fun sort(toSort: List<BlockVector3>, clipboard: Clipboard): List<List<BlockVector3>> {
        return listOf(toSort)
    }
}